cmd/skopeo/layers.go

```diff
@@ -70,9 +78,6 @@ var layersCmd = cli.Command{
 		defer dest.Close()
 
 		for _, digest := range blobDigests {
-			if !strings.HasPrefix(digest, "sha256:") {
-				digest = "sha256:" + digest
-			}
```

Comment:
This would change behavior. Unless @runcom or someone else knows that this is no longer necessary, we need to preserve this, probably above `ParseDigest` above.